src/analyz/mod.rs

```diff
@@ -635,6 +628,14 @@ impl<'tcx> mir::visit::MutVisitor<'tcx> for SubstConstTys<'tcx> {
         self.super_constant(constant, location);
     }
 }
+*/
+fn subst_const_tys<'tcx>(
+    tcx: TyCtxt<'tcx>,
+    substs: ty::subst::SubstsRef<'tcx>,
+    body: &mut Body<'tcx>,
+) {
+    // FIXME remove
+}
```

Comment:
Can this be removed already? It seems like it's called once, for side effects, and it doesn't have any.

Reply:
Well, all the tests are passing, so I guess it's safe to remove.  `subst_const_tys` was a workaround for a rustc bug, which seems to be fixed in the new nightly.